.gitattributes

```diff
@@ -0,0 +1,6 @@
+/tests                      export-ignore
+/.gitattributes             export-ignore
+/.gitignore                 export-ignore
+/phpunit.xml                export-ignore
+/CHANGELOG.md               export-ignore
```

Comment:
can we actually leave the CHANGELOG.md file and README.md could be useful if someone installs and needs to look at some documentation